test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperDescriptorFactoryTest.cs

```diff
@@ -138,10 +138,12 @@ public static TheoryData OutputElementHintData
                                 attributes: Enumerable.Empty<TagHelperAttributeDescriptor>(),
                                 requiredAttributes: Enumerable.Empty<string>(),
                                 tagStructure: default(TagStructure),
-                                designTimeDescriptor: new TagHelperDesignTimeDescriptor(
-                                    summary: null,
-                                    remarks: null,
-                                    outputElementHint: "strong"))
+                                designTimeDescriptor: new TagHelperDesignTimeDescriptor
+                                {
+                                    Summary = null,
```

Comment:
No need to set these values to `null`. Just don't set them. Same applies for other instances.